Frameworks/UIKit/UIButton.mm

```diff
@@ -184,7 +184,7 @@ - (void)_initUIButton {
     _contentVerticalAlignment = UIControlContentVerticalAlignmentCenter;
     _contentHorizontalAlignment = UIControlContentHorizontalAlignmentCenter;
 
-    __block UIButton* weakSelf = self;
+    __weak UIButton* weakSelf = self;
```

Comment:
>UIButton [](start = 11, length = 8)

unrelated:  in UIView.mm, line 758, is it also another misusage?

    // Subscribe to the XAML node's input events
    __block UIView* weakSelf = self;